participant/tests/test_api.py

```diff
@@ -294,3 +296,46 @@ def test_participant_scores_debrief_behaviour_responses(self):
                     behaviour,
                     response.data["phishing_emails"][0]["participant_behaviour"],
                 )
+
+
+class OrganizationQuerysetTests(APITestCase):
+    def setUp(self):
+        self.organization = OrganizationFactory(name="this_test_organization")
+        self.organization2 = OrganizationFactory(name="this_test_organization 2")
+        self.organization3 = OrganizationFactory(name="this_test_organization 3")
+        self.user = UserFactory(
+            username="user_with_organization", organization=self.organization
+        )
+        self.superuser = UserFactory(
+            organization=self.organization, is_superuser=True, username="test_superuser"
+        )
+        self.user_without_organization = UserFactory(
+            username="user_without_organization"
+        )
```

Comment:
you are not going to need username for these users if you tweak the `UserFactory` as suggested below

Reply:
as above. Just for test specific readability